content/docs/user-guide/scripting/script-canvas/editor-reference/shortcuts.md

```diff
@@ -12,31 +12,31 @@ The **Script Canvas Editor** includes the following keyboard and mouse shortcuts
+| **Ctrl+Shift+H** | Ungroup | Ungroups the currently selected group. |
+| **Ctrl+Number_Key** | Create bookmark | Creates a bookmark out of the current view and assigns it to the specified number key. If you choose a number that is already assigned to a bookmark or a bookmark-enabled group, you are prompted to reassign the existing bookmark. For more information about bookmarks, see [Adding Bookmarks in Script Canvas](/docs/user-guide/scripting/script-canvas/editor-reference/nodes/organizing/adding-bookmarks). For information about enabling groups as bookmarks, see [Grouping Nodes](/docs/user-guide/scripting/script-canvas/editor-reference/nodes/organizing/grouping-nodes). |
 | Number_Key | Jump to bookmark | Jumps to the bookmark location associated with the key that is pressed. |
-| Ctrl+Plus Sign (+) | Zoom in | Zooms the graph in. |
-| Ctrl+Minus Sign (-) | Zoom out | Zooms the graph out. |
-| Ctrl+Shift+Up Arrow | Zoom to selection | Centers the view on the nodes that are currently selected. |
-| Ctrl+Shift+Down Arrow | Show entire graph | Centers the entire graph into the current display. Zooms out as much as possible to display all nodes. |
-| Ctrl+Shift+Left Arrow | Show start of chain | Centers the view on the nodes that do not have any input connections, and are connected to the selected node through their output connections. |
-| Ctrl+Shift+Right Arrow | Show end of chain | Centers the view on the nodes that do not have any output connections, and are connected to the selected node through their input connections. |
-| Ctrl+K, Ctrl+C | Disable selected nodes | Disables the selected nodes. When a node is disabled, its color changes to gray. Disabled nodes remain visible in the Editor, but are not run at runtime. |
-| Ctrl+K, Ctrl+U | Enable selected nodes | Enables the selected nodes. |
+| **Ctrl+Plus Sign (+)** | Zoom in | Zooms the graph in. |
+| **Ctrl+Minus Sign (-)** | Zoom out | Zooms the graph out. |
+| **Ctrl+Shift+Up Arrow** | Zoom to selection | Centers the view on the nodes that are currently selected. |
+| **Ctrl+Shift+Down Arrow** | Show entire graph | Centers the entire graph into the current display. Zooms out as much as possible to display all nodes. |
+| **Ctrl+Shift+Left Arrow** | Show start of chain | Centers the view on the nodes that do not have any input connections, and are connected to the selected node through their output connections. |
+| **Ctrl+Shift+Right Arrow** | Show end of chain | Centers the view on the nodes that do not have any output connections, and are connected to the selected node through their input connections. |
+| **Ctrl+K**, **Ctrl+C** | Disable selected nodes | Disables the selected nodes. When a node is disabled, its color changes to gray. Disabled nodes remain visible in the Editor, but are not run at runtime. |
+| **Ctrl+K**, **Ctrl+U** | Enable selected nodes | Enables the selected nodes. |
```

Comment:
```suggestion
| **Ctrl+C** | Disable selected nodes | Disables the selected nodes. When a node is disabled, its color changes to gray. Disabled nodes remain visible in the Editor, but are not run at runtime. |
| **Ctrl+U** | Enable selected nodes | Enables the selected nodes. |
```
Ctrl+K wasn't toggling nodes for me.  I could only get Ctrl+K and Ctrl+U to work sporadically.

Reply:
I assume that's what was meant by the comma then. Thanks!